<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiBoundary</class>
 <widget class="QMainWindow" name="UiBoundary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UiBoundary</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>480</width>
      <height>305</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>480</width>
      <height>272</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>480</width>
      <height>305</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="SmartPram">
     <attribute name="title">
      <string>Smart Pram</string>
     </attribute>
     <widget class="QPushButton" name="Button_Vugge">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>M Vugge</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_Uro">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>M Uro</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_Automatisk">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Automatisk</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_Indstillinger">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>SystemIndstillinger</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Temp_LCD">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>130</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="Temperatur">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperatur:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Degress">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>140</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>grader</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="Sound_LCD">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>180</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Procent">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>190</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
     <widget class="QLabel" name="Sound">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>190</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Lyd:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Temp_Alarm">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TEMP ALARM</string>
      </property>
     </widget>
     <widget class="QLabel" name="Vugge_Ikon">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Vugge er ikke Aktiv</string>
      </property>
     </widget>
     <widget class="QLabel" name="Uro_Ikon">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Uro er ikke Aktiv</string>
      </property>
     </widget>
     <widget class="QLabel" name="Auto_Ikon">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Automatisk er ikke Aktiv</string>
      </property>
     </widget>
     <widget class="QLabel" name="Sound_Alarm">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>LYD ALARM</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Indstillinger">
     <attribute name="title">
      <string>Indstillinger</string>
     </attribute>
     <widget class="QPushButton" name="Save_settings">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Gem</string>
      </property>
     </widget>
     <widget class="QSlider" name="Lydmax_Slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="Lydmax_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Lyd Max</string>
      </property>
     </widget>
     <widget class="QLabel" name="Vuggetid_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Vuggetid</string>
      </property>
     </widget>
     <widget class="QLabel" name="Urotid_Label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Urotid</string>
      </property>
     </widget>
     <widget class="QSlider" name="Vuggetid_Slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="Urotid_Slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="Urotid_vaerdi">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lydmax_vaerdi">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSlider" name="Tempmax_Slider">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>110</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000000</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="Tempmax_Label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>80</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Temp Max</string>
      </property>
     </widget>
     <widget class="QSlider" name="Tempmin_Slider">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="Tempmin_Label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>140</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Temp Min.</string>
      </property>
     </widget>
     <widget class="QLabel" name="Tempmax_vaerdi">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>80</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Tempmin_vaerdi">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>140</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Vuggetid_vaerdi">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>57</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_cancel">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuller</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Button_default">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>200</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
